src/main/webapp/app/guided-tour/guided-tour.service.ts

```diff
@@ -313,7 +313,7 @@ export class GuidedTourService {
      */
     public get preventBackdropFromAdvancing(): boolean {
         if (this.currentTour) {
-            return this.currentTour && (this.currentTour.preventBackdropFromAdvancing ? this.currentTour.preventBackdropFromAdvancing : false);
+            return this.currentTour && (this.currentTour.preventBackdropFromAdvancing ? this.currentTour.preventBackdropFromAdvancing : true);
```

Comment:
Nit: This basically means that the default value for `preventBackdropFromAdvancing` is true.
It could be a problem in future if this default value needs to be managed by the service and not the domain object.

One idea would be to make the GuidedTour a class:

```
export class GuidedTour {
    /** Identifier for tour */
    settingsKey: string;
    /** Steps fo the tour */
    steps: TourStep[];
    /** The given function will be called when tour is skipped */
    skipCallback?: (stepSkippedOn: number) => void;
    /** The given function will be called when tour is completed */
    completeCallback?: () => void;
    /** Minimum size of screen in pixels before the tour is run, if the tour is resized below this value the user will be told to resize */
    minimumScreenSize?: number;
    /**
     * Prevents the tour from advancing by clicking the backdrop.
     * This should only be set if you are completely sure your tour is displaying correctly on all screen sizes otherwise a user can get stuck.
     */
    preventBackdropFromAdvancing = true;
}
```